packages/clerk-js/src/ui/elements/Avatar.tsx

```diff
@@ -91,23 +89,41 @@ export const Avatar = (props: AvatarProps) => {
   );
 };
 
+const avatarColors = ['#6C47FF', '#5BC5EF', '#FBD486', '#BEF972', '#FF8F8F'];
```

Comment:
Let's merge this one and figure out a way to make the avatar colors customizable via `appearance` with a new PR.